src/ts/helpers/heuristics.ts

```diff
@@ -69,6 +69,13 @@ export function isSecure(entry: WaterfallEntry) {
   return entry.name.indexOf("https://") === 0;
 }
 
+export function isPush(entry: WaterfallEntry) {
+  const harEntry = entry.rawResource;
+  // WebPageTest got 1 if an asset was pushed, Browsertime = true
+  console.log(harEntry["_was_pushed"]);
+  return ( harEntry["_was_pushed"] === true || Number(harEntry["_was_pushed"]) === 1) ? true : false;
```

Comment:
Aha, so maybe explicit check against "1" would be clearer. But very minor nitpick.